Microsoft.DotNet.Interactive/IKernel.cs

```diff
@@ -16,5 +16,7 @@ public interface IKernel : IDisposable
         IObservable<IKernelEvent> KernelEvents { get; }
 
         Task<IKernelCommandResult> SendAsync(IKernelCommand command, CancellationToken cancellationToken);
+
+        void SetDefaultRendering();
```

Comment:
There's no generalization of default rendering that makes sense across all possible kernels, and the concept may not be something we even want to commit to for the two existing kernels. Let's keep the implementations specific to the cases where we understand what this means and keep the interface simpler for now.

Reply:
Then move this back to be specific to the C# kernel?  That's where we originally started which seems at odds with how I interpreted [this](https://github.com/dotnet/try/pull/357#discussion_r311773451) comment, specifically, I read this:

> maybe we should add an overload for FSharpKernel.

As

> move this method to KernelBase

But to properly support `CompositeKernel`, the method needed to be on `IKernel`, instead.

Reply:
Sorry. I meant that we should have two methods, one for `CSharpKernel` and one for `FSharpKernel`. Their implementations are different and I don't know if this method will make sense for all kernels.

Reply:
Added commit 03e8866, let me know what you think.